charts/posthog/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 3.2.0
+version: 3.3.0
```

Let's keep the previous version. We bump versions via merging PRs with appropriate labels (will handle it).

```suggestion
version: 3.2.0
```